core/toolbox/toolbox.js

```diff
@@ -68,7 +68,7 @@ Blockly.Toolbox = function(workspace) {
    * The unique id for this component.
    * @type {string}
    */
-  this.id = Blockly.utils.genUid();
+  this.id = 'toolbox';
```

Comment:
I think I would prefer either this was updated to `componentId` or we update the comment to say this is used to register the item with the component manager.

Reply:
I've updated the jsdoc to reference the `ComponentManager`.